rpc/test/helpers.go

```diff
@@ -100,8 +100,8 @@ func GetGRPCClient() core_grpc.BroadcastAPIClient {
 }
 
 // StartTendermint starts a test tendermint server in a go routine and returns when it is initialized
-func StartTendermint(app abci.Application) *nm.Node {
-	node := NewTendermint(app)
+func StartTendermint(app abci.Application) (*nm.Node, func()) {
```

Comment:
Can we create `StopTendermint()` function which removes a config directory?

```go
// node/node.go
func (n *Node) Config() *cfg.Config {
	return n.config
}

//  rpc/test/helpers.go
func StopTendermint() {
	node.Stop()
	node.Wait()
	os.RemoveAll(node.Config().RootDir)
}
```

Reply:
If you don't mind, I will do it in an ensuing PR

Reply:
Sure 👍